dynamic_rest/renderers.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from rest_framework import renderers
+
+
+class DynamicJSONRenderer(renderers.JSONRenderer):
+
+  def render(self, data, accepted_media_type=None, renderer_context=None):
```

Comment:
BTW, I copied this to Apollo to integrate it with the V3 stuff, and it worked pretty seamlessly. Yay! The only thing I had to change was to check for `_model` so it wouldn't try to modify options responses and batch create responses (see comment below).